src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -784,4 +786,36 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase() {
             <error>pub</error> const C: u32 = 10;
         }
     """)
+
+    @Test
```

Comment:
You don't need `@Test` attributes here. It's enough that the method names start with `test`.